app/pages/dashboard.tsx

```diff
@@ -82,39 +81,23 @@ const Dashboard = ({
       },
     });
   };
+  
+  const openIntakeBanner = useFeature('open_intake_alert').value || {};
+  const closedIntakeBanner = useFeature('closed_intake_alert').value || {}; 
 
   return (
     <Layout session={session} title="Connecting Communities BC">
       <div>
         <section>
           <h1>Dashboard</h1>
           {!openIntake && (
-            <IntakeAlert openTimestamp={nextIntake?.openTimestamp} />
+            <DynamicAlert dateTimestamp={nextIntake?.openTimestamp} text={closedIntakeBanner.text}
+            variant={closedIntakeBanner.variant} includeLink 
+            displayOpenDate = {closedIntakeBanner.displayOpenDate}/>        
           )}
-          {openIntake ? (
-            <p>
-              Review of applications will begin on{' '}
-              {DateTime.fromISO(closeTimestamp, {
-                locale: 'en-CA',
-                zone: 'America/Vancouver',
-              }).toFormat('MMMM dd, yyyy, ttt')}
-              . You can edit draft and submitted applications until this date.
-            </p>
-          ) : (
-            <div>
-              <p>Applications are currently not being accepted.</p>
-              <p>
-                Please check the{' '}
-                <Link
-                  href="https://www2.gov.bc.ca/gov/content/governments/connectivity-in-bc/20601/20601-63737"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  program webpage
-                </Link>{' '}
-                for updates.
-              </p>
-            </div>
```

Comment:
This should not be deleted. The dynamic alert should be added above, but should not replace this

Reply:
changes rolled back